app/src/main/java/a/a/a/Ix.java

```diff
@@ -239,7 +239,6 @@ private void writeAndroidxRoomService(XmlBuilder application) {
 
     private void writeAndroidxStartupInitializationProvider(XmlBuilder application) {
         var initializers = Set.of(
-                new Pair<>(builtInLibraryManager.containsLibrary(BuiltInLibraries.ANDROIDX_EMOJI2), "androidx.emoji2.text.EmojiCompatInitializer"),
```

Comment:
Why you removed this library?

Reply:
its the first thing i did cause this didnt allow me to investigate more so i remove it from being initialised, more i digged the net about it and i found out that this will make the created apps more smoother for example scroling contents in recycler and etc more i didnt see any impact to projects rather than that user still can use the lib as he want

Reply:
i u revert this, app will crash without any log just leave it as is for now we can investigate it later

Reply:
@hasrat-ali still want this to be reverted ?